dist/src/main/config/broker.standalone.yaml.template

```diff
@@ -424,6 +424,14 @@
           # This setting can also be overridden using the environment variable ZEEBE_BROKER_DATA_BACKUP_AZURE_BASEPATH
           # basePath:
 
+          # Configures which authentication method is used for connecting to Azure. Can be either 'auto' or be left empty.
+          # Choosing 'auto' means that the Azure client uses application default credentials which automatically
+          # discovers appropriate credentials from the runtime environment: https://learn.microsoft.com/en-us/entra/identity/managed-identities-azure-resources/how-manage-user-assigned-managed-identities?pivots=identity-mi-methods-azp
```

Comment:
:thought_balloon: Should we rather link to https://learn.microsoft.com/en-us/java/api/com.azure.identity.defaultazurecredential?view=azure-java-stable? This at least explains how the credentials are discovered, precedence rules etc.